src/sentry/eventstream/kafka/backend.py

```diff
@@ -27,11 +41,8 @@
 
 logger = logging.getLogger(__name__)
 
-
-class PostProcessForwarderType(str, Enum):
-    ERRORS = "errors"
-    TRANSACTIONS = "transactions"
-    ISSUE_PLATFORM = "search_issues"
+if TYPE_CHECKING:
```

Comment:
Wasn't there a discussion about not using TYPE_CHECKING with models @wedamija ? I'm not familiar enough with the problem to know the best alternate solution though.

Reply:
I don't think we really decided anything conclusively there, was just a random slack convo, so no need to change behaviour for now